dali/python/nvidia/dali/ops.py

```diff
@@ -1027,5 +1083,31 @@ def register_gpu_op(name):
 from nvidia.dali.external_source import ExternalSource
 ExternalSource.__module__ = __name__
 
-_load_ops()
+class CompoundOp:
+    def __init__(self, op_list):
+        self.ops = op_list
+
+    def __call__(self, *inputs, **kwargs):
+        for op in self.ops:
+            inputs = op(*inputs, **kwargs)
+            kwargs = {}
+            if isinstance(inputs, _DataNode):
+                inputs = (inputs,)
+
+        return inputs[0] if len(inputs) == 1 else inputs
+
+def Compose(op_list):
```

Comment:
Is this visible in our supported operations table? Can you paste a screenshot?

Reply:
I think it may deserve a bigger and more complete example.

Reply:
For now it's marked as experimental. I think the next step would be to send it to the proponent of the whole Compose affair to play with and. Once we get some feedback and suggestions for improvements, we can improve the functionality and provide comprehensive documentation - or, conversely, remove it if we decide it's not necessary after all.

Reply:
It is not listed in `Support Table`.
I think you need ```
global _cpu_ops
    _cpu_ops = _cpu_ops.union({'DLTensorPythonFunction'})
    global _gpu_ops
    _gpu_ops = _gpu_ops.union({'DLTensorPythonFunction'})
```

Reply:
Also how it handles CPU to GPU memory transfer?

Reply:
I added the handling of transfers.
As for the support table, it's a bit harder, since this is not an operator in native sense and doesn't have a schema.

Reply:
Do we need a schema to make it a part of the table?

Reply:
Unless I hardcode it into the table generator - then yes. When I tried just putting the operator there using set union, Sphinx failed with `No schema registered for operator Compose`.